config/dictionaries/os_version.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "id": "2.0",
```

Comment:
Should this be Semvery? like 2.0.0? Or are we only documenting minor release changes?